test/shlibloadtest.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2016-2017 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
Surely, two copyright lines means dual copyright, no?  As far as I know, that's valid.